README.md

```diff
@@ -73,112 +72,113 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.62 - August 2022 Update
+### 0.63 - September 2022 Update
 
-In this release, we focused on releasing three new PowerToys.
+In this release, we focused on stability and improvement.
 
 **Highlights**
 
-- New utility: Screen Ruler is a quick and easy way to measure pixels on your screen.
-- New utility: Quick Accent is an easy way to write letters with accents. Thanks [@damienleroy](https://github.com/damienleroy)!
-- New utility: Text Extractor works like Snipping Tool, but copies the text out of the selected region using OCR and puts it on the clipboard. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
-- PowerToy Run ships with a new Plugin letting you search in past query results. Thanks [@jefflord](https://github.com/jefflord)!
+- QuickAccent contains a new setting to select a language. This should reduce the number of accented characters a user needs to pick from. Thanks [@damienleroy](https://github.com/damienleroy)!
+- Reduced installer and installed sizes by sharing the Windows App SDK, VC++ redistributable and PowerToys Interop runtime files between utilities.
```

Comment:
We should also reference this work will help us ultimately remove the UAC on install. Also say we will be working with getting .NET runtime work to do the same.

Reply:
Also we could directly call out disk and file reduction here. They are pretty massive

Reply:
Disk? I suppose you mean installation _file_ and/or installation?
Curious to see how much it'll be :)

Reply:
Disk and install file is what I was referring to. The installer is much smaller and removed one dependency and our footprint on disk is much more reduced.